pkg/packagekit/wix/wix.go

```diff
@@ -76,6 +83,22 @@ func WithDocker(image string) WixOpt {
 	}
 }
 
+func WithUI() WixOpt {
+	return func(wo *wixTool) {
+		wo.ui = true
+
+	}
+}
+
+func WithFile(name string, content []byte) WixOpt {
+	return func(wo *wixTool) {
+		if wo.extraFiles == nil {
```

Comment:
I was a little surprised that this nil check was needed, since it seemed like the ' value for the type `[]extraFile` would be an empty slice, and thus `append`-able. I was playing around with this [here](https://play.golang.org/p/ZtFXCvxF66L), and I _think_ that we don't need the nil check, does that seem correct to you?

Reply:
I think you're right. `append` totally works on nil.